systemtests/tests/stresstest/cloneandcopykernelcode

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#   BAREOS® - Backup Archiving REcovery Open Sourced
+#
+#   Copyright (C) 2022-2022 Bareos GmbH & Co. KG
+#
+#   This program is Free Software; you can redistribute it and/or
+#   modify it under the terms of version three of the GNU Affero General Public
+#   License as published by the Free Software Foundation and included
+#   in the file LICENSE.
+#
+#   This program is distributed in the hope that it will be useful, but
+#   WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#   Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program; if not, write to the Free Software
+#   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+#   02110-1301, USA.
+
+set -e
+set -o pipefail
+
+# clone the linux kernel code from github and make 10 copies of it
+# it would amount to 75GB of data, and around 1M files
+
+mkdir kernelcopies
+cd kernelcopies
+git clone https://www.github.com/torvalds/linux
```

Comment:
Is this test enabled by default?
Probably we want to execute it only if explicitly asked for.